<launch>
    <group>
        <node pkg="autotarget" type="safety" name="SAFETY" output="screen" ns="/drone1">
            <param name="namespace" value="/drone1"/>
            <param name="use_sim_time"  value="true" />
            <param name="target_system_id" value="1" />
        </node>
    </group>
    <group>
        <node pkg="autotarget" type="safety" name="SAFETY" output="screen" ns="/drone2">
            <param name="namespace" value="/drone2"/>
            <param name="use_sim_time"  value="true" />
            <param name="target_system_id" value="2" />
        </node>
    </group>
</launch>
